Makefile

```diff
@@ -117,7 +118,7 @@ docker: build_ui build-agent-linux build-collector-linux build-query-linux
 	cp -r jaeger-ui-build/build/ cmd/query/jaeger-ui-build
 	docker build -t $(DOCKER_NAMESPACE)/jaeger-cassandra-schema plugin/storage/cassandra/ ; \
 	for component in agent collector query ; do \
-		docker build -t $(DOCKER_NAMESPACE)/jaeger-$$component cmd/$$component ; \
+		docker build -t $(DOCKER_NAMESPACE)/jaeger-$$component:${DOCKER_TAG} cmd/$$component ; \
```

Comment:
That part is done on the `upload-to-docker.sh`, for builds running on `master`:
https://github.com/uber/jaeger/blob/master/travis/upload-to-docker.sh#L5